docs/source/modules.rst

```diff
@@ -0,0 +1,4 @@
+Project modules
```

Comment:
Consider removing this as the modules are not entered here yet. Should it prove necessary with this section we can add it at a later point.

Reply:
Removed.